lms/djangoapps/shoppingcart/models.py

```diff
@@ -824,6 +824,73 @@ def generate_pdf_invoice(self, course, course_price, quantity, sale_price):
 
         return pdf_buffer
 
+    def __unicode__(self):
+        return "company: {}".format(self.company_name)
+
+
+INVOICE_TRANSACTION_STATUSES = (
+
+    ('started', 'started'),
+    ('completed', 'completed'),
+    ('cancelled', 'cancelled'),
+)
+
+
+class InvoiceTransaction(TimeStampedModel):
+    """
+         This table capture all the information needed to support "InvoicingTransactions"
```

Comment:
Maybe add a comment indicating that this represents both credits (positive amounts) and refunds (negative amounts).